### ISSUE TITLE
@category doc over class props, causes constructor to be categorized as "Other"

### ISSUE THREAD

**Issue by charlesLoder** [typedoc2md/typedoc-plugin-markdown#661]
### What package is the bug related to?

typedoc-plugin-markdown

### Describe the issue

This may not be a bug, per se, but the ergonomics of it feel a little off.

If the properties of a class have the `@category` tag, then the `constructor` is no longer given the heading "Constructors"; instead, it is under the heading "Other."

One possible workaround is to give the `constructor` the `@category` tag of `@category Constructor`, but it will cause the "Constructor" heading to alphabetically come after other categories (e.g. "Color"). So this isn't a fix.

See [minimal example](https://stackblitz.com/edit/vitejs-vite-ajhmog?file=src%2Findex.ts) and run `npm run docs`, which also outputs the Typedoc json.
The `animalWithoutCategories.ts` outputs the `constructor` under the right heading, and `animalWithCategories.ts` outputs the `constructor` under the heading of "Other".

### TypeDoc configuration

```json
{
  "plugin": ["typedoc-plugin-markdown"],
  "out": "./docs",
  "entryPoints": ["src/index.ts"]
}
```

### Expected behavior

I expect that the Constructor would retain it's own unique heading, and be at the top of the page under headings like "Implements", which is not affected by alphabetical ordering. I would expect this given the importance of a constructor in a class.

**Comment by tgreyuk** [comment]
Thanks - will review and fix with next release.

**Comment by tgreyuk** [comment]
Should be fixed in typedoc-plugin-markdown@4.2.4.

**Comment by charlesLoder** [comment]
It works, thanks!
